mmeds/tests/metadata/test_error_metadata.py

```diff
@@ -0,0 +1,96 @@
+from unittest import TestCase
+import mmeds.validate as valid
+import mmeds.config as fig
+import sys
+import traceback as tb
+from pathlib import Path
+
+# Set as True to see formatted error output
+VERBOSE = False
+
+class ErrorTests(TestCase):
+    def test_a_generated_error_files(self):
+        """
+        Test all the error files generated by scripts/generate_test_tsv.py
+        """
+        
+        top_dir = Path('/home/adamcantor22//mmeds-meta/test_files/validation_files/')
+        sub_directories = ['blank_column_tests', 'na_column_tests', 'other_column_tests', 'number_column_tests', 'date_column_tests']
+
+        # Get all sub directories to analyze
+        total_directories = []
+        for directory in sub_directories:
+            total_directories.append(top_dir / directory / 'subject')
+            total_directories.append(top_dir / directory / 'specimen')
+            # total_directories.append(top_dir / 'misc_tests')
+
+        good_subjects = valid.validate_mapping_file(fig.TEST_SUBJECT_SHORT, 'Short_Study', 'subject', None, 'human')[2]
+        for directory in total_directories:
+            for test_file in directory.glob('*.tsv'):
+                exceptions_covered = True
+
+                # Generate headers for each file output
+                if VERBOSE:
+                    length = len(test_file.name)
+                    bar_size = 70 
+                    bar = ''
+                    count = int((bar_size - length) / 2)
+                    for i in range(count):
+                        bar += '='
+                    bar += ' ' + test_file.name + ' '
+                    for i in range(count):
+                        bar += '='
+                    print(bar) 
```

Comment:
These print blocks should be moved into a separate function that can be called on (test_file.name). Having all the extra code directly in the test obscures what the test is actually doing